### ISSUE TITLE
Options aren't passed unless you pass a string to protocol

### ISSUE THREAD

**Issue by reboxer** [porsager/pws#4]
So i found this module which is exactly what i need, but it looks like the ws `options` won't be passed correctly unless you pass a parameter for `protocol`, i found out this while testing, i kept getting "401" when trying to connect and after that, i decided to check the headers of the request, the "authentication" header was missing, until i added a empty and not empty string as the second parameter (protocol) and the header was included in the request.

request that didn't work:
```js
const ws = Pws("ws://localhost:9090", Websocket, {
    headers: {
        authentication: "test"
    }
});
```

request that worked:
```js
const ws = Pws("ws://localhost:9090", "", Websocket, {
    headers: {
        authentication: "test"
    }
});
```

**Comment by porsager** [comment]
Ah, good find, thanks a lot :) - I've released a fix with v4.0.1

**Comment by reboxer** [comment]
Thanks for the fast fix and reply, it works 👍
